wikipedia_cleanup/data_processing.py

```diff
@@ -158,7 +161,9 @@ def get_data(
             mapped_filters.append(changes_and_filters[1])
     all_changes = itertools.chain.from_iterable(all_changes)
     merge_filter_stats_into(mapped_filters, filters)
-    return pd.DataFrame([change.__dict__ for change in all_changes])
+    res_df = pd.DataFrame([change.__dict__ for change in all_changes])
```

Comment:
No need to build a list :)
```suggestion
    res_df = pd.DataFrame(change.__dict__ for change in all_changes)
```